source/StellarObject.cpp

```diff
@@ -155,6 +156,34 @@ double StellarObject::Distance() const
 
 
 
+void StellarObject::UpdateDistanceVisibility(const Ship *ship)
+{
+	if(trueDistanceInvisible < 0.)
```

Comment:
just wondering, would it make sense to have a 0 distance visible thing? ~~!truesmth would be more clear~~

Reply:
~it would make sense if you want to troll somebody~

Reply:
```suggestion
	if(!trueDistanceInvisible)
```